Gemfile

```diff
@@ -49,10 +49,8 @@ group :development do
   gem 'capistrano3-puma',   require: false
   gem 'web-console',           '>= 3.3.0'
   gem 'listen',                '~> 3.0.5'
-  gem 'spring'
-  gem 'spring-watcher-listen', '~> 2.0.0'
 end
 
 group :production do
-  gem 'pg', '~> 0.18'
```

Comment:
Do you know what compatibility this pg version has with the postgres database version? If it's not far backwards compatible we could have an issue and need to update the database version on prod before deploying.